graphql-spring-boot-autoconfigure/src/main/java/graphql/kickstart/autoconfigure/web/servlet/GraphQLWebAutoConfiguration.java

```diff
@@ -301,15 +302,13 @@ public GraphQLConfiguration graphQLServletConfiguration(
       @Autowired(required = false) GraphQLResponseCacheManager responseCacheManager,
       @Autowired(required = false) AsyncTaskDecorator asyncTaskDecorator,
       @Autowired(required = false) @Qualifier("graphqlAsyncTaskExecutor") Executor asyncExecutor) {
-    long asyncTimeout =
-        graphQLServletProperties.getAsyncTimeout() != null
-            ? graphQLServletProperties.getAsyncTimeout()
-            : asyncServletProperties.getTimeout();
+    long asyncTimeout = Optional.ofNullable(graphQLServletProperties.getAsyncTimeout()) //
+        .orElse(asyncServletProperties.getTimeout()).toMillis();
     return GraphQLConfiguration.with(invocationInputFactory)
         .with(graphQLInvoker)
         .with(graphQLObjectMapper)
         .with(listeners)
-        .with(graphQLServletProperties.getSubscriptionTimeout())
+        .with(graphQLServletProperties.getSubscriptionTimeout().toMillis())
```

Comment:
Same potential `null` issue as with async timeout.

Reply:
I try to use Optional.ofNullable() to deal with it

Reply:
Refactoring with PropertyMapper might be better,But I need your help

Reply:
I would not mix the PropetyMapping refactoring part into this PR, as it is not really related to the feature implemented here.

Reply:
I agree with your suggestion. I'm just proposing possible solutions